src/token.ts

```diff
@@ -74,6 +78,36 @@ export async function fetchAppToken(
     };
 }
 
+/**
+ * Crowdin Agent Authentication
+ */
+
+interface FetchAgentTokenArgs extends FetchAppTokenArgs {
+    agentId?: number;
```

Comment:
is it a valid state that `agentId` can be undefined and we still use `fetchAgentToken`?